python/doc/bibliography.rst

```diff
@@ -123,6 +123,8 @@ Bibliography
 .. [koay2006] Koay C.G., Basser P.J.,
     *Analytically exact correction scheme for signal extraction from noisy magnitude MR signals*,
     Journal of magnetics Resonance 179, 317-322, 2006.
+.. [koch2019] Koch, H., & Bopp, G. P., *Fast and exact simulation of multivariate normal and Wishart random variables with box constraints*,
+    arXiv preprint arXiv:1907.00057, 2019.
```

Comment:
```suggestion
```
We're no longer using the algorithm proposed by Koch et al following @regislebrun's comment:

https://github.com/openturns/openturns/pull/2028/files#r860588932